examples/example-rust-vm/src/lib.rs

```diff
@@ -2,62 +2,25 @@ extern crate evmc_vm;
 
 use evmc_vm::*;
 
-extern "C" fn execute(
-    instance: *mut ffi::evmc_instance,
-    context: *mut ffi::evmc_context,
-    rev: ffi::evmc_revision,
-    msg: *const ffi::evmc_message,
-    code: *const u8,
-    code_size: usize,
-) -> ffi::evmc_result {
-    let execution_ctx = unsafe {
-        ExecutionContext::new(
-            msg.as_ref().expect("tester passed nullptr as message"),
-            context.as_mut().expect("tester passed nullptr as context"),
-        )
-    };
-    let is_create = execution_ctx.get_message().kind == ffi::evmc_call_kind::EVMC_CREATE;
+#[evmc_declare_vm("ExampleRustVM", "evm")]
```

Comment:
Is `evm` the version here? If so, we should use a usual version string because otherwise it is confusing, e.g. "0.1.0".

Reply:
that's the capabilities set. It is a `", "` -separated list of capabilities, which at the moment includes `"ewasm"` and `"ewasm"`. I need to document somewhere visible.

Reply:
The version is the crate version. I may add an option to specify a crate-independent version, as I have figured out how to make this possible.

Reply:
Well these should be rustdoc documented in the `evmc-declare` crate as well as with a comment in this example file.